### ISSUE TITLE
Unable to set up

### ISSUE THREAD

**Issue by iCHAIT** [osuosl/timesync-node#246]
I was trying to set it up on my local machine.

`npm install` worked without a hitch and all dependencies got installed.

Now, when i run `TIMESYNC_AUTH_MODULES='["password"]' npm run devel`

It gives me the following error in the console - 

```sh
> timesync@0.0.0 devel /Users/chaitanyagupta/Desktop/timesync-node
> nodemon --harmony ./src/app.js

[nodemon] 1.8.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node --harmony ./src/app.js`
[nodemon] app crashed - waiting for file changes before starting...

```

What's possible going wrong?

cc @LyonesGamer

**Comment by iCHAIT** [comment]
I tried running `npm start` and it gave me the following error - 

```
> timesync@0.0.0 start /Users/chaitanyagupta/Desktop/timesync-node
> node --harmony ./src/app.js


npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/5.7.0/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v5.7.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! timesync@0.0.0 start: `node --harmony ./src/app.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the timesync@0.0.0 start script 'node --harmony ./src/app.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the timesync package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node --harmony ./src/app.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs timesync
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls timesync
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/chaitanyagupta/Desktop/timesync-node/npm-debug.log
```

**Comment by iCHAIT** [comment]
@LyonesGamer 

Okay, so I was finally able to set up `timesync-node` on my machine.

The problems that I was facing in this particular issue were resolved by #247 and #249.

Thanks
